### PR TITLE
Tanta 99

### PR DESCRIPTION
**Pull request by amcpherson**

### CODE REVIEW

File: tantalus/settings.py [hunk 1]
```diff
@@ -96,7 +96,8 @@
         'NAME': os.environ.get('TANTALUS_POSTGRESQL_NAME'),
         'USER': os.environ.get('TANTALUS_POSTGRESQL_USER'),
         'PASSWORD': os.environ.get('TANTALUS_POSTGRESQL_PASSWORD'),
-        'HOST': '10.9.215.82',
+        #'HOST': '10.9.215.82',
```
Comment (amcpherson) [inline tantalus/settings.py]: Dont check in commented code

File: tantalus/models.py [hunk 2]
```diff
@@ -499,11 +527,17 @@ class SequenceDataset(models.Model):
         (UNUSABLE, 'Alignments are not usable'),
     )
 
-    reference_genome = models.CharField(
-        max_length=50,
-        # TODO: foreign key or update choices
-        #choices=reference_genome_choices,
-        default=UNALIGNED,
+    #temp = models.CharField(
+    #    max_length=50,
```
Comment (amcpherson) [inline tantalus/models.py]: dont check in commented out code

File: tantalus/models.py [hunk 3]
```diff
@@ -420,6 +420,34 @@ class SequenceFileInfo(models.Model):
...
+
+    history = HistoricalRecords()
+
+    #HG19 = 'HG19'
+    #HG18 = 'HG18'
+    #MM10 = 'MM10'
+    #UNALIGNED = 'UNALIGNED'
+    #UNUSABLE = 'UNUSABLE'
+    #
+    #reference_genome_choices = (
```
Comment (amcpherson) [inline tantalus/models.py]: dont commit commented out code

File: tantalus/migrations/0082_auto_20181130_2330.py [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+    for r in recs:
+        v = vocab.objects.get(reference_genome=r.temp)
+        r.reference_genome=v
+        if (0 == (r.id % 113)):
+            print("----- %s; %s;" % (r.id,r.reference_genome.id))
+        r.save()
+
+
+def do0(apps, schema_editor):
+    pass
```
Comment (amcpherson) [inline tantalus/migrations/0082_auto_20181130_2330.py]: what is this?

File: tantalus/models.py [hunk 5]
```diff
@@ -420,6 +420,34 @@ class SequenceFileInfo(models.Model):
...
+    #    (MM10, 'Mouse Genome 10'),
+    #    (UNALIGNED, 'Not aligned to a reference'),
+    #    (UNUSABLE, 'Alignments are not usable'),
+    #)
+
+    reference_genome = models.CharField(
+        max_length=50,
+        null=True,
+        unique=True,
+        #default=UNALIGNED, #not necessary
```
Comment (amcpherson) [inline tantalus/models.py]: remove commented code

File: tantalus/models.py [hunk 6]
```diff
@@ -420,6 +420,34 @@ class SequenceFileInfo(models.Model):
...
+    #
+    #reference_genome_choices = (
+    #    (HG19, 'Human Genome 19'),
+    #    (HG18, 'Human Genome 18'),
+    #    (MM10, 'Mouse Genome 10'),
+    #    (UNALIGNED, 'Not aligned to a reference'),
+    #    (UNUSABLE, 'Alignments are not usable'),
+    #)
+
+    reference_genome = models.CharField(
```
Comment (amcpherson) [inline tantalus/models.py]: change field to be called 'name'

File: tantalus/migrations/0082_auto_20181130_2330.py (hunk 4 above)
Comment (svn779) [reply tantalus/migrations/0082_auto_20181130_2330.py]: Empty function, required to rollback migration, without it rollback fails.

**Pull request closed without merging**